### DISCUSSION TITLE
Using AlpineJS without Livewire

### DISCUSSION THREAD

**Discussion by citricguy** [livewire/livewire#6395] [1. Help] [answered]
I have a few pages that do not contain livewire components. I've noticed that these pages do not load Alpine assets properly. If I add a Livewire component to the page, the alpine code starts to work.

Do I need to manually inject the livewire components (ie., @livewireStyles and @livewireScripts) to get this to work now that AlpineJS is bundled with Livewire?

Is there a way to have livewire to include the AlpineJS assets automatically on every page, even when a Livewire component is not used?

Adding `{{\Livewire\Livewire::forceAssetInjection()}}` to the top of my blade template that uses AlpineJS works, but this really doesn't feel right.

**Comment by joydeep-bhowmik** [comment]
same problem here

**Reply by citricguy** [reply]
I see it now in the docs.

```
AlpineJS is bundled with Livewire
Because Alpine is bundled with Livewire's JavaScript assets, you must include on every page you wish to use Alpine. Even if you're not using Livewire on that page.
```

https://livewire.laravel.com/docs/installation

There are a few options there in how to resolve this, or how to include on every page.

**Comment by PhiloNL** [comment, marked as answer]
Add `\Livewire\Livewire::forceAssetInjection()` to your `AppServiceProvider` should do the trick.

```php
class AppServiceProvider extends ServiceProvider
{
    /**
     * Register any application services.
     *
     * @return void
     */
    public function register()
    {
        \Livewire\Livewire::forceAssetInjection();
    }
}
```
